packages/mattermost-redux/src/actions/admin.test.js

```diff
@@ -0,0 +1,1272 @@
+        assert.ok(teamAnalytics[TestHelper.basicTeam.id][Stats.POST_PER_DAY]);
+    });
+
+    it('getUsersPerDayAnalytics', async () => {
+        nock(Client4.getBaseRoute()).
+            get('/analytics/old').
+            query(true).
+            times(2).
+            reply(200, [{name: '2017-06-18', value: 2}, {name: '2017-06-16', value: 47}, {name: '2017-06-12', value: 4}, {name: '2017-06-08', value: 55}, {name: '2017-06-07', value: 2}, {name: '2017-06-06', value: 1}, {name: '2017-06-05', value: 2}, {name: '2017-06-04', value: 13}, {name: '2017-06-02', value: 1}, {name: '2017-05-31', value: 3}, {name: '2017-05-30', value: 4}, {name: '2017-05-29', value: 3}, {name: '2017-05-26', value: 40}, {name: '2017-05-25', value: 26}, {name: '2017-05-24', value: 43}, {name: '2017-05-23', value: 3}]);
+
+        await Actions.getUsersPerDayAnalytics()(store.dispatch, store.getState);
+        await Actions.getUsersPerDayAnalytics(TestHelper.basicTeam.id)(store.dispatch, store.getState);
+
+        const state = store.getState();
+        const request = state.requests.admin.getAnalytics;
+        if (request.status === RequestStatus.FAILURE) {
+            throw new Error('getAnalytics request failed');
+        }
+
+        const analytics = state.entities.admin.analytics;
+        assert.ok(analytics);
+        assert.ok(analytics[Stats.USERS_WITH_POSTS_PER_DAY]);
+
+        const teamAnalytics = state.entities.admin.teamAnalytics;
+        assert.ok(teamAnalytics);
+        assert.ok(teamAnalytics[TestHelper.basicTeam.id]);
+        assert.ok(teamAnalytics[TestHelper.basicTeam.id][Stats.USERS_WITH_POSTS_PER_DAY]);
+    });
+
+    it('overwritePlugin', async () => {
+        const data1 = fs.createReadStream('tests/setup.js');
+        const data2 = fs.createReadStream('tests/setup.js');
+        const testPlugin = {id: 'testplugin', webapp: {bundle_path: '/static/somebundle.js'}};
+
+        nock(Client4.getBaseRoute()).
+            post('/plugins', (body) => {
+                return !body.match(/Content-Disposition: form-data; name="force"\r\n\r\ntrue\r\n/);
+            }).
+            reply(200, testPlugin);
+        await Actions.uploadPlugin(data1, false)(store.dispatch, store.getState);
+
+        nock(Client4.getBaseRoute()).
+            post('/plugins', (body) => {
+                return body.match(/Content-Disposition: form-data; name="force"\r\n\r\ntrue\r\n/);
+            }).
+            reply(200, testPlugin);
+        await Actions.uploadPlugin(data2, true)(store.dispatch, store.getState);
+
+        const state = store.getState();
+        const request = state.requests.admin.uploadPlugin;
+        if (request.status === RequestStatus.FAILURE) {
+            throw new Error('uploadPlugin request failed err=' + request.error);
+        }
+    });
+
+    it('uploadPlugin', async () => {
+        const testFileData = fs.createReadStream('packages/mattermost-redux/test/assets/images/test.png');
```

Comment:
Another path that needed to change since Node doesn't look at any of the import aliasing that's now set up.